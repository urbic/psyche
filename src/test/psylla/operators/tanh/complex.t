0. 1. complex tanh ?
