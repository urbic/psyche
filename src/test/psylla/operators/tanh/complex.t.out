0.0+1.557407724654902*i
